backend/apis/leads/update-lead.api.js

```diff
@@ -86,7 +86,7 @@ app.patch("/api/leads/:id", (req, res, next) => {
 
     const leadServicesChanged = atLeastOne(req.body, "leadServices");
 
-    if (leadServicesChanged && fullLead.leadServices.length > 0) {
+    if (leadServicesChanged || fullLead.leadServices.length > 0) {
```

Comment:
This is a duplicate of https://github.com/JustUtahCoders/comunidades-unidas-internal/pull/397.

You messed up the branches when creating PRs - this is something to continue practicing.

Reply:
Luckily this one wasn't a git oops. I re-corrected the error while working on this. Forgot I'd already fixed it in the other pending PR. Should I switch it back still?